go/ai/gen.go

```diff
@@ -109,10 +122,12 @@ const (
 	RoleTool Role = "tool"
 )
 
+// A TextPart is a [Part] consisting of plain text.
```

Comment:
I think a TextPart can be unexported just like MediaPart.

Reply:
Done.